src/sentry/monitors/constants.py

```diff
@@ -7,3 +7,6 @@
 
 # Format to use in the issue subtitle for the missed check-in timestamp
 SUBTITLE_DATETIME_FORMAT = "%b %d, %I:%M %p"
+
+# maximum value for incident + recovery thresholds to be set
+MAX_THRESHOLD = 720
```

Comment:
Can we add a comment for what turning this knob does. And maybe point out that lowering it would invalidate some monitor configurations